### PR TITLE
Add SpawnWorld srv

### PR DESCRIPTION
**Pull request by Tadinu**

### CODE REVIEW

**Review by yuokamoto — COMMENTED** [review]
Ca't we use SpawnEntity srv?

**Comment by Tadinu** [comment]
We could but that would bring a bit of complication into the handling of the `SpawnEntitySrvCallback`, which would have to do 2 things:
* Detect whether the Entity is a world or robot model, for which another parameter might have to be added into the Entity service struct
* Branch off the handling to call `SpawnEntity` & `SpawnWorld`
Since technically since Entity and World are 2 different concepts, for sake of clarity and maintenance, I'd suggest we make them separate services

**Comment by Tadinu** [comment]
We could but that would bring a bit of complication into the handling of the `SpawnEntitySrvCallback`, which would have to do 2 things:
* Detect whether the Entity is a world or robot model, for which another parameter might have to be added into the Entity service struct
* Branch off the handling to call `SpawnEntity` & `SpawnWorld`
Since technically `Entity` and `World` are 2 different concepts, for sake of clarity and maintenance, I'd suggest we make them separate services

**Comment by yuokamoto** [comment]
Isn't it simple to use geometry_msgs/Pose instead of EntityState?

**Comment by Tadinu** [comment]
Ah right, so if users would not need to provide a given name for the to-be-spawned world instance for whatever needs they might have, like to order the world displacement or just to query its transform, so a name property might be essential in that case I guess.

**Comment by Tadinu** [comment]
Ah right, so if users would need to provide a given name for the to-be-spawned world instance, like to order the world displacement or just to query its transform, so a name property might be essential in that case I guess.

**Comment by Tadinu** [comment]
Ah right, so if users need to provide a given name for the to-be-spawned world instance, like to order the world displacement or just to query its transform, so a name property might be essential in that case I guess.

File: srv/SpawnWorld.srv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+string world_model                # Entity world model name
+ue_msgs/EntityState state         # World given name + initial pose
```
Comment (yuokamoto) [inline srv/SpawnWorld.srv:2]:
```suggestion
geometry_msgs/Pose pose       # World given name + initial pose
```
geometry_msgs/Pose pose

Comment (Tadinu) [reply srv/SpawnWorld.srv:2]: Ok aside from the pose, let it have a separate string world_name also

Comment (Tadinu) [reply srv/SpawnWorld.srv:2]: Ok aside from the pose, let it have a separate `string name` also

**Review by yuokamoto — APPROVED** [review]
please update ros1 branch as well

**Pull request merged**